definitions/json/State.json

```diff
@@ -14,12 +14,20 @@
     "DisplayOrder": 2,
     "TitleDisplay": "# Case Number: ${ethosCaseReference} </br> <h4>${flagsImageAltText}</h4>"
   },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "AWAITING_SUBMISSION_TO_HMCTS",
+    "Name": "AWAITING_SUBMISSION_TO_HMCTS",
+    "Description": "Draft ET1 updates and this state carried till case submission ",
+    "DisplayOrder": 3,
+    "TitleDisplay": "# Case Number: ${ethosCaseReference} </br> <h4>${flagsImageAltText}</h4>"
```

Comment:
I would remove TitleDisplay property so that the default one is used. We aren't going to have the ethosCaseReference and flagsImageAltText for a draft case